src/main/splitting/tasks_tests_handler.py

```diff
@@ -101,6 +101,19 @@ def __get_user_folder_name_from_path(file: str) -> str:
 
 
 def run_tests(path: str) -> str:
+    """
+    Run tests on all code snapshots in the data for the task.
+    Note: the enum class TASK (see consts.py file)  must have the task key.
+    It also must match the name of the folder with test files in the resources/tasks_tests.
+
+    For example, if your task has key [my_key], you should add a new value into TASK const with value [my_key]
+    and add a new folder [my_key] with input and output files for tests in the resources/tasks_tests folder.
+
+    The test result is an array containing values for all tasks from the TASK enum class.
+    If the code snapshot is incorrect, then the value -1 is specified.
+    To deserialize this array of ratings, use the function unpack_tests_results from splitting.py.
+    TODO: should we add a function to get the tests results by the current task?
```

Comment:
I thought we already have had such a function, but now i'm not sure
Maybe we shouldn't run tests for all tasks now since we improved codetracker and it's quite impossible to have a solution with the wrong task

Reply:
We run tests only for the current task, but store array for all tasks with 0 value for other tasks.

Reply:
Also, we have a function to unpack tests results into an array, but not for getting score for the current task